fdbserver/EncryptKeyProxy.actor.cpp

```diff
@@ -173,7 +228,13 @@ struct EncryptKeyProxyData : NonCopyable, ReferenceCounted<EncryptKeyProxyData>
 
 		EncryptBaseCipherDomainIdKeyIdCacheKey cacheKey = getBaseCipherDomainIdKeyIdCacheKey(domainId, baseCipherId);
 		baseCipherDomainIdKeyIdCache[cacheKey] =
-		    EncryptBaseCipherKey(domainId, domainName, baseCipherId, baseCipherKey, true);
+		    EncryptBaseCipherKey(domainId,
+		                         domainName,
+		                         baseCipherId,
+		                         baseCipherKey,
+		                         // CipherKey if NOT latest, don't expire or need-refresh
```

Comment:
non-latest cipher key may still be revokable?

Reply:
Thought about it and couldn't find a good reason to do it. 

Non-latest ciphers are technically the CipherKeys which were used in past and have been rotated by a newer incarnation by KMS. They can never be used to encrypt newer data, however, still needs to be present to allow decrypting already stored data. 
The `expired` rotated ciphers would be the Cipher which is `destroyed` and it is not used to encrypt any data, technically, KMS should never respond with such a key, as CipherKey lifecycle management is beyond FDB (no way for FDB to enforce it, hence, not worth handling?)

Reply:
Thanks for the explanation. I was confused about the intent of revocable key.